src/meta/service/src/network.rs

```diff
@@ -184,6 +184,7 @@ impl RaftNetwork<LogEntry> for Network {
                     mes = resp.into_inner();
                     // clean up
                     last_err = None;
+                    break;
```

Comment:
How about `return xxxx` directly?

Reply:
Good, we don't need to care about `last_err` then.